Integrations/Gmail/Gmail.py

```diff
@@ -165,8 +237,17 @@ def get_email_context(email_data, mailbox):
     body = demisto.get(email_data, 'payload.body.data')
     body = body.encode('ascii') if body is not None else ''
     parsed_body = base64.urlsafe_b64decode(body)
+    context_email = {}  # type: Dict
+    if email_data.get('internalDate') is not None:
+        utc, delta_in_seconds = utc_extract(str(headers.get('date', '')))
+        base_time = datetime.fromtimestamp((int(str(email_data.get('internalDate'))[:10]))) + timedelta(seconds=delta_in_seconds)
+        day = days_of_the_week(int(base_time.isoweekday()))  # gets day of week from date
+        base_time = day + ", " + time_reformat(str(base_time)) + " " + utc
+    # return_error(str(headers.get('date', ''))+" ### "+base_time)
+    else:
+        base_time = None
```

Comment:
Not sure you want to set this to None. Might want to fallback to original behaviour in this case. Also, add a warning as if I understand correctly it shouldn't happen.